pages/_app.js

```diff
@@ -7,21 +7,42 @@ function MyApp({ Component, pageProps }) {
 
     {}
   );
+  const [spendingValues, setSpendingValues] = useLocalStorage(
+    "spendingValues",
+
+    []
+  );
 
   function addBankAccount(bankAccount) {
     setBankAccount((previousBankAccount) => {
       return {
         ...previousBankAccount,
         name: bankAccount.name,
-        kontostand: bankAccount.kontostand,
+        kontostand: parseFloat(bankAccount.kontostand),
```

Comment:
I am surprised that here "kontostand" is not in English. I would change it to English since it is not part of the user interface shown, but code component.

Reply:
Yes, it´s on my to-do-list for the next user-story. The functionality was more important to me for now.